xet-tools.rb

```diff
@@ -11,7 +11,7 @@ class XetTools < Formula
   end
 
   on_linux do
-    url "https://github.com/xetdata/xet-tools/releases/download/v0.7.1/xet-v0.7.1-linux-x86_64.tar.gz"
+    url "https://github.com/xetdata/xet-tools/releases/download/v0.7.1/xet-v0.7.1-linux.tar.gz"
```

Comment:
Not sure this will fix the version detection. Should we add an explicit version field as well (or instead)?